### PR TITLE
LPS-70122 SF

### PR DESCRIPTION
**Pull request by inacionery**

### CODE REVIEW

**Comment by inacionery** [comment]
ci:retest

**Comment by inacionery** [comment]
ci:retest

**Comment by marcellustavares** [comment]
@inacionery, will be reviewing this pull today. Thx

File: portal-workflow-definition-web/src/main/java/com/liferay/portal/workflow/definition/web/internal/portlet/action/AddWorkflowDefinitionMVCActionCommand.java [hunk 1]
```diff
@@ -31,5 +44,31 @@
...
+
+		String content = ParamUtil.getString(actionRequest, "content");
+		Map<Locale, String> titleMap = LocalizationUtil.getLocalizationMap(
+			actionRequest, "title");
+
+		if (Validator.isNull(content)) {
+			throw new WorkflowDefinitionFileException();
+		}
+
+		WorkflowDefinitionManagerUtil.deployWorkflowDefinition(
```
Comment (marcellustavares) [inline portal-workflow-definition-web/src/main/java/com/liferay/portal/workflow/definition/web/internal/portlet/action/AddWorkflowDefinitionMVCActionCommand.java]: Please use reference to WorkflowDefinitionManager, instead

**Pull request merged**